Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+//
+//  DetailngCodebaseViewController.swift
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+import SnapKit
+
+class DetailingCodebaseViewController: BaseViewController {
```

Comment:
p3;

어딘가에서 상속받는 경우가 아니라면 `final` 키워드를 붙여주세요 !
```suggestion
final class DetailingCodebaseViewController: BaseViewController {
```

Reply:
> p3;
> 
> 어딘가에서 상속받는 경우가 아니라면 `final` 키워드를 붙여주세요 !

daf8eb4 넵 감사합니다~!